src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -471,9 +472,12 @@ def log_step(self, message, header=False):
             header (bool, optional): whether to log a header line before the message. Defaults to
                 False.
         """
+        elapsed = time() - self._test_step_time
+        self._test_step_time = time()
```

Comment:
Minor - I think it's better to call `time()` only once here to avoid subtle differences. E.g. 1 second off.
```suggestion
        _time = time()
        elapsed = _time - self._test_step_time
        self._test_step_time = _time
```

Reply:
Done.